Extension/src/Debugger/configurationProvider.ts

```diff
@@ -252,6 +232,64 @@ class CppConfigurationProvider implements vscode.DebugConfigurationProvider {
         return config && config.type ? config : null;
     }
 
+    private convertEnvFileToEnvironmentMap(config: vscode.DebugConfiguration, folder: vscode.WorkspaceFolder): void {
```

Comment:
Should this be called resolveEnvFile ? because you are resolving it to the environment object.